phoenix-core/src/it/java/org/apache/phoenix/end2end/ViewMetadataIT.java

```diff
@@ -186,7 +190,8 @@ public void testCreateViewMappedToExistingHbaseTableWithNamespaceMappingEnabled(
 
             // test for a view that is in non-default schema
             {
-                HTableDescriptor desc = new HTableDescriptor(TableName.valueOf(NS, TBL));
+                tableName = TableName.valueOf(NS, TBL);
```

Comment:
Why did we move the declarations out of the block?  I'm assuming the declarations were local so we would be less likely to have same variable reused with wrong values scenario.

Reply:
Hmm I think I pushed this change accidentally. I will revert it